CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
```

Comment:
Could you just remove this empty line, and the ones below `### Added` and `### Changed`? These lines are causing a conflict in the merge.

Other than that, everything is fine :)